source/puppet/4.1/reference/resources_package_windows.markdown

```diff
@@ -102,6 +102,8 @@ It's a good idea to use the hash notation for file path arguments since they may
 install_options => [ { 'INSTALLDIR' => "${packagedir}" } ]
 {% endhighlight %}
 
+I'm not sure this ^ is necessary as puppet should quote arguments with spaces automatically.
+
```

Comment:
because it needs to be `"/INSTALLDIR=""C:\Program Files\somewhere"""` for Chocolatey to pass `INSTALLDIR="C:\Program Files\somewhere"` to the underlying installer.